src/controllers/document.download.controller.ts

```diff
@@ -74,15 +74,15 @@ const prefixFilename = (contentDisposition: string ): string => {
 const showErrorPage = (httpStatus: number, res: Response) => {
   const { UNAUTHORIZED, FORBIDDEN, NOT_FOUND } = HttpStatusCodes;
   switch (httpStatus) {
-    case UNAUTHORIZED:
-    case FORBIDDEN:
-    case NOT_FOUND: {
-      const { HEADING_NOT_FOUND, MESSAGE_NOT_FOUND } = DownloadErrorMessages;
-      return renderFileError(res, HEADING_NOT_FOUND, MESSAGE_NOT_FOUND);
-    }
-    default: {
-      return res.render(Templates.ERROR);
-    }
+  case UNAUTHORIZED:
+  case FORBIDDEN:
+  case NOT_FOUND: {
```

Comment:
This indentation doesn't look right to me, especially with the 2 braces vertically aligned below. does it complain if we have normal indentation like this 
`switch (httpStatus) {
    case UNAUTHORIZED:`

Reply:
Needed to specify for switch statements.